package.json

```diff
@@ -0,0 +1,37 @@
+{
+  "devDependencies": {
+    "@fewlines/eslint-config": "3.1.2",
+    "@types/jest": "28.1.1",
+    "@types/node": "17.0.40",
+    "@types/pg": "8.6.5",
+    "@types/uuid": "8.3.4",
+    "@typescript-eslint/eslint-plugin": "5.27.0",
+    "@typescript-eslint/parser": "5.27.0",
+    "eslint": "8.17.0",
+    "eslint-config-prettier": "8.5.0",
+    "eslint-plugin-import": "2.26.0",
+    "eslint-plugin-prettier": "4.0.0",
+    "jest": "28.1.0",
+    "jest-fetch-mock": "3.0.3",
+    "lerna": "5.0.0",
+    "mock-http": "1.1.0",
+    "prettier": "2.6.2",
+    "supertest": "6.2.3",
+    "ts-jest": "28.0.4",
+    "ts-node": "10.8.1",
+    "typescript": "4.7.3"
+  },
+  "name": "node-web-libraries",
+  "dependencies": {
+    "@fwl/database": "file:packages/database",
+    "@fwl/logging": "file:packages/logging",
+    "@fwl/migration": "file:packages/migration",
+    "@fwl/oauth2": "file:packages/oauth2",
+    "@fwl/tracing": "file:packages/tracing",
+    "@fwl/web": "file:packages/web"
+  },
+  "private": true,
+  "workspaces": [
+    "packages/*"
+  ]
```

Comment:
This adds everything to every `/packages/<package>`, right?

Reply:
not exactly, this tells `yarn` that if you go to `packages/tracing` and you run `yarn install`, yarn will look at the root `yarn.lock` and not behave as if you were in a classical folder

Reply:
I thought it was more like a default setup and we could overwrite it in `packages/tracing/package.json` but no, ok. 😃